.github/CODEOWNERS

```
@@ -43,3 +34,18 @@ store/google/en
 
 ## Utils ##
 store/google/Publisher
+
+## These workflows have joint ownership ##
+.github/workflows/build.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/build-beta.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/cleanup-rc-branch.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/release.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/version-auto-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/version-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+
+# Shared ownership for version bump automation
+src/App/Platforms/Android/AndroidManifest.xml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.Autofill/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.Extension/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.ShareExtension/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/App/Platforms/iOS/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
```

```suggestion
src/App/Platforms/Android/AndroidManifest.xml
src/iOS.Autofill/Info.plist
src/iOS.Extension/Info.plist
src/iOS.ShareExtension/Info.plist
src/App/Platforms/iOS/Info.plist
```

These have to have no owner in order for our automatic PR process to work between the bitwarden-devops-bot and the github-bot.  Sorry, I wasn't clear in my first message.

---

Ah, gotcha. Thanks!